site/cypress/e2e/gradeables/gradeable_results.spec.js

```diff
@@ -0,0 +1,51 @@
+
+
+describe('Testing functionality of Autograder Results', () => {
+    beforeEach(() => {
+        cy.visit(['sample','gradeable','closed_team_homework']);
+        cy.login('student');
+
+    });
+
+    it('Should display confetti', () => {
+        cy.get('#confetti_canvas').should('have.css', 'display').and('not.match',/block/);
```

Comment:
I don't have a working VM to test this locally, is there a reason to test the CSS 'display' instead of using .should('be.visible') or .should('exist')  (Similar to William's previous comment)

Reply:
Yep, this needs to be fixed before this PR can be merged.

Reply:
The reason I didn't change it is because should('be.visible') doesn't work on confetti canvas because its ancestor element has position:fixed. The way it's displayed in js code is by changing the display property, so querying this specifically is better for testing the code.